src/atomicui/organisms/SearchBox/SearchBox.tsx

```diff
@@ -104,12 +104,13 @@ const SearchBox: React.FC<SearchBoxProps> = ({
 					exact,
 					undefined,
 					TriggeredByEnum.PLACES_SEARCH,
-					action
+					action,
+					isNLChecked
 				);
 				setIsSearching(false);
 			}, 200);
 		},
-		[mapRef, setIsSearching, search]
+		[mapRef, setIsSearching, search, isNLChecked]
```

Comment:
Don't see how this is getting translated to `isNLSearchEnabled` in `search` method. What am I mising? :\